controllers/activities.js

```diff
@@ -0,0 +1,91 @@
+const models = require('../models')
+
+const getAllActivities = async (request, response) => {
+  try {
+    const activities = await models.Activities.findAll()
+
+    return response.send(activities)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
+
+const getActivitiesByMoon = async (request, response) => {
+  try {
+    const { moonId } = request.params
+
+    const foundMoon = await models.Activities.findOne({
+      where: { moonId }
+    })
+
+    return foundMoon
+      ? response.send(foundMoon)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
+
+const getActivitiesByMonth = async (request, response) => {
+  try {
+    const { monthId } = request.params
+
+    const foundMonth = await models.Activities.findOne({
+      where: { monthId }
+    })
+
+    return foundMonth
+      ? response.send(foundMonth)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Error')
+  }
+}
```

Comment:
If I am getting activities by month shouldn't I get more than one? I would expect to see ALL activities associated with that month.